app/uk/gov/hmrc/agentaccesscontrol/connectors/mtd/RelationshipsConnector.scala

```diff
@@ -40,18 +41,24 @@ class RelationshipsConnector @Inject()(@Named("agent-client-relationships-baseUr
   extends HttpAPIMonitor {
   override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
 
-  def relationshipExists(arn: Arn, mtdItId: MtdItId)
-                       (implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Boolean] = {
-    monitor("ConsumedAPI-AgentClientRelationships-CheckMtdItId-GET") {
-      httpGet.GET[HttpResponse](relationshipUrl(arn, mtdItId).toString)
+  def relationshipExists(arn: Arn, identifier: TaxIdentifier)
+                        (implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Boolean] = {
+    val (serviceName, clientType, clientId) = identifier match {
+      case _ @ MtdItId(mtdItId) => ("HMRC-MTD-IT", "MTDITID", mtdItId)
+      case _ @ Vrn(vrn) => ("HMRC-MTD-VAT", "VRN", vrn)
+    }
+
+    val relationshipUrl =
+      new URL(baseUrl,
+        s"/agent-client-relationships/agent/${arn.value}/service/$serviceName/client/$clientType/$clientId").toString
+
+    monitor(s"ConsumedAPI-AgentClientRelationships-Check$clientType-GET") {
```

Comment:
Capitalisation of this metric has now changed for the MtdItId type from "MtdItId" to "MTDITID". Not sure if this has an impact on the metrics shown on the dashboards or not. It might need updating at the same time as adding in the new VRN metric.

Reply:
Separate PR will be raised against grafana-dashboards with updated labels. I think we need to keep the previous label in order to preserve the existing data.